pkg/interface/src/views/apps/publish/components/MetadataForm.tsx

```diff
@@ -47,30 +39,32 @@ const ResetOnPropsChange = (props: { init: FormSchema; book: string }) => {
 
 
 export function MetadataForm(props: MetadataFormProps) {
-  const { api, book } = props;
+  const { book } = props;
   const { metadata } = props.association || {};
+  const api = useApi();
 
   const initialValues: FormSchema = {
     name: metadata?.title,
     description: metadata?.description,
   };
 
-
   const onSubmit = async (
     values: FormSchema,
     actions: FormikHelpers<FormSchema>
   ) => {
     try {
       const { name, description } = values;
-      await api.metadata.metadataAdd(
+      await api.poke(metadataAdd(
+        window.ship,
         "publish",
         props.association.resource,
+        // TODO there should be a path here
```

Comment:
@liam-fitzgerald or @tacryt-socryp the `metadataAdd` poke requires a path. Which do I use?